### PR TITLE
[UI] Dashboard to System Dashboard #1633

### LINKED ISSUE #1633: [UI] Dashboard to System Dashboard
**Current Behavior**
The Dashboard page describes Meshery components and system status. This page is named "Dashboard".

**Desired Behavior**
This page should be renamed and updated to reflect more of what it is.

**Implementation**
1. This page should be renamed to "System Dashboard".
1. The two separate Prometheus and Grafana cards should be combined into a single "Metrics" card.

**Mockups**
Frame 1-4

**Acceptance Tests**
1. UI should like similar to the mockup with cards rearranged.
1. Dashboard component should be renamed to SystemDashboard.

### PR DESCRIPTION
**Pull request by karkranikhil**
**Description** I have fixed the heading from Dashboard to System Dashboard. Apart from text fix the cards alignments by giving the box-shadow

This PR fixes #1633

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by karkranikhil** [comment]
image

File: ui/components/DashboardComponent.js [hunk 1]
```diff
@@ -75,13 +75,17 @@ const styles = (theme) => ({
   },
   card: {
     height: '100%',
+    boxShadow: '0px 0px 10px grey',
```
Comment (kushthedude) [inline ui/components/DashboardComponent.js:78]: Please remove shadows, it is not looking well, the shadow in mockup was to demonstrate. Material-UI already has a minimal shadow component which is quite neat and tidy.

File: ui/components/DashboardComponent.js [hunk 2]
```diff
@@ -515,16 +519,16 @@ class DashboardComponent extends React.Component {
...
-            <Grid item xs={12} md={6}>
+            <Grid item xs={12} md={6} className={classes.gridPadding}>
               {self.showCard('Adapters', showAdapters)}
             </Grid>
-            <Grid item xs={12} md={6}>
+            <Grid item xs={12} md={6} className={classes.gridPadding}>
               {self.showCard('Grafana', showGrafana)}
             </Grid>
-            <Grid item xs={12} md={6}>
+            <Grid item xs={12} md={6} className={classes.gridPadding}>
```
Comment (kushthedude) [inline ui/components/DashboardComponent.js:531]: See the above review

File: ui/components/Header.js [hunk 3]
```diff
@@ -94,7 +94,7 @@ class Header extends React.Component {
                 </Hidden>
                 <Grid item xs container alignItems="center" className={classes.pageTitleWrapper}>
                   <Typography color="inherit" variant="h5" className={classes.pageTitle}>
-                    {title}
+                    {title === 'Dashboard' ? 'System Dashboard' : title}
```
Comment (kushthedude) [inline ui/components/Header.js:97]: Can this be renamed in the page route rather than a hack in the components?

**Review by kushthedude — COMMENTED** [review]
Hi @karkranikhil, please see some suggested changes

File: ui/components/DashboardComponent.js (hunk 2 above)
Comment (karkranikhil) [reply ui/components/DashboardComponent.js]: Okay i will revert these changes.

File: ui/components/DashboardComponent.js (hunk 1 above)
Comment (karkranikhil) [reply ui/components/DashboardComponent.js]: Okay i will revert these changes.

File: ui/components/Header.js (hunk 3 above)
Comment (karkranikhil) [reply ui/components/Header.js]: I have fixed it on page routes

**Review by leecalcote — COMMENTED** [review]
Much appreciated, @karkranikhil! 

Will you also have a look at the second portion of the request in #1633 that asks for a consolidation of two of the cards into one?

Thanks for this. 👍

**Comment by karkranikhil** [comment]
image

**Comment by leecalcote** [comment]
Oooh, nice!

**Review by kushthedude — CHANGES_REQUESTED** [review]
Please rebase your PR to current master

**Review by leecalcote — CHANGES_REQUESTED** [review]
@karkranikhil this is looking really good. Just was the issue called for.

We have merge conflict that I'm hoping you can take a look at - https://github.com/layer5io/meshery/pull/1635/conflicts

Thanks!

**Comment by leecalcote** [comment]
@karkranikhil please signal if you're able to continue this work. In the meantime, @daksh-sagar would you like to carry this forward?

@karkranikhil in the event that you don't see this note in time, please know that your efforts here are much appreciated. And that there's more where this came from. 😉

**Comment by daksh-sagar** [comment]
@leecalcote yes, I'll work on this 🙂

**Comment by Nikhil-Ladha** [comment]
Closing this as the follow-up PR can be found here #1668

**Pull request closed without merging**